src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import duke.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    public TaskList(ArrayList<Task> storage) {
+        this.tasks = storage;
+    }
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void addTask(Task task) {
+        this.tasks.add(task);
+    }
+
+    public Task getTask(int i) {
+        return tasks.get(i);
+    }
+
+    public Task removeTask(int i) {
+        return this.tasks.remove(i);
+    }
```

Comment:
Since there are no other similarly-named local variables as `tasks` in some of these, perhaps you could consider removing `this` from `this.tasks`.